deis-tests-parallel/manifests/deis-tests-parallel-pod.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: deis-tests
+  namespace: deis
+  labels:
+    heritage: deis
+spec:
+  restartPolicy: Never
+  containers:
+  - name: deis-e2e
+    image: quay.io/deisci/deis-e2e:canary
+    imagePullPolicy: Always
+    command: ["ginkgo"]
+    args: ["-slowSpecThreshold=120.00", "-nodes=30", "-noisyPendings=false", "-progress", "tests/"]
```

Comment:
@vdice  the `-stream` option IIRC made the output really noisy. Instead by turning off the verbose mode and using `GinkgoWriter` it will output the correct log messages for tests failures and nothing else. 

@arschles yes except that I figured we might want to make these configurable for people who are running them on their own cluster. I didnt want to bake all of that into the image.